lib/beaker/hypervisor/vsphere_helper.rb

```diff
@@ -11,15 +11,24 @@ def initialize vInfo
                                        :insecure => true
   end
 
-  def self.load_config(dot_fog = '.fog')
-    default = get_fog_credentials(dot_fog)
-
+  def self.load_config(dot_fog = '.fog', vsphere_instance = "")
     vsphere_credentials = {}
-    vsphere_credentials[:server] = default[:vsphere_server]
-    vsphere_credentials[:user]   = default[:vsphere_username]
-    vsphere_credentials[:pass]   = default[:vsphere_password]
 
-    return vsphere_credentials
+    if (vsphere_instance.blank?)
```

Comment:
As we talked about just now @jcoconnor it's probably better if we just load in the `vcenter_instances` config. here too. So the old code would still be here, you'd just have an extra line after 20 that's something like:
```
vsphere_credentials[:vcenter_instances] = default[:vcenter_instances]
```

Doing it this way is better b/c we have the same interface for `load_config` and we let the caller handle how to deal with the `vcenter_instances` config. info, which only beaker-vcloud seems to care about right now. Plus this shouldn't break anything since it's an additive feature. @kevpl can you confirm?